integration_tests/pages/page.ts

```diff
@@ -138,6 +142,21 @@ export default abstract class Page {
     })
   }
 
+  shouldContainRadioButtons(options: Array<{ label: string; value: string }>): void {
+    cy.get('.govuk-radios__item').each((radioButtonElement, radioButtonElementIndex) => {
+      const option = options[radioButtonElementIndex]
```

Comment:
Is it worth flipping this so that we can use the `.eq` method you used before here? E.g. `options(or items).forEach((option, optionIndex) => cy.get('.govuk-radios__item').eq(optionIndex)...`?